PhysicsTools/NanoAOD/python/taus_cff.py

```diff
@@ -9,6 +9,11 @@
 
 ##################### User floats producers, selectors ##########################
 
+# Original DeepTau v2p5 in 12_4_X doesn't include WPs in MINIAOD
+# Import thresholds here to define WPs manually from raw scores
+from copy import deepcopy
+from RecoTauTag.RecoTau.tools.runTauIdMVA import WORKING_POINTS_v2p5
+working_points_v2p5 = deepcopy(WORKING_POINTS_v2p5)
 
 finalTaus = cms.EDFilter("PATTauRefSelector",
```

Comment:
Overlooked issue: final taus selection (cut) should contain in its "big OR" also loosest WP of deepTau VS jets